src/Ratchet/Http/HttpRequestParser.php

```diff
@@ -56,7 +56,7 @@ public function onMessage(ConnectionInterface $context, $data)
      */
     public function isEom($message)
```

Comment:
Ideally, the function calls should be fixed instead. Sooner or later, a "string" type hint will be added for $message, and the problem will strike us back.